lms/djangoapps/branding/migrations/0001_initial.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
```

Comment:
In general adding our own migrations to an an existing app would not be a problem on production. The migrations are sequenced <and> named such that there should never be a conflict. The ghost migration issue should only affect dev environments when one goes back and forth between origin and upstream branches and I'm not at all sympathetic to doing anything different there. That having been said, I did add an initial migration to an upstream app, which I think would cause a file level conflict, so I've moved it to its own app. Thanks for noticing.